### DISCUSSION TITLE
Add option to uninvert controls when building backwards.

### DISCUSSION THREAD

**Discussion by karst** [OpenRCT2/OpenRCT2#17969] [Feature requests]
I think it'd be a very nice option to be able to build backwards the same way as forwards without the controls being inverted. Then have an option available to switch it back to RCT2's behaviour. I heard that this is already an option that's available in vanilla RCT3, though I haven't checked this myself.

**Comment by TheClaudeQC** [comment]
it is like that in RCT3, I did play it by myself.

Sometimes, I confuse RCT3 and RCT2 each times I recently RCT3
